src/Explorer/Panes/PublishNotebookPaneAdapter.tsx

```diff
@@ -161,19 +158,27 @@ export class PublishNotebookPaneAdapter implements ReactAdapter {
         this.content,
         this.isLinkInjectionEnabled
       );
-      if (response.data) {
-        NotificationConsoleUtils.logConsoleMessage(ConsoleDataType.Info, `Published ${name} to gallery`);
+
+      const data = response.data;
+      if (data) {
+        if (data.pendingScanJobIds?.length > 0) {
+          NotificationConsoleUtils.logConsoleInfo(
+            `Content of ${this.name} is currently being scanned for illegal content. It will not be available in the public gallery until the review is complete (may take a few days).`

```

Comment:
>(may take a few days). [](start = 161, length = 22)

this paranthesised text looks little informal. Can you check with Jessica?  Can we word it something like : "It can even take upto a few days"

Reply:
This is just a placeholder text. I'm thinking once this is deployed we can play around and ask for Deborah's help.